packages/deployments/contracts/package.json

```diff
@@ -3,8 +3,9 @@
   "version": "2.0.0-alpha.1",
   "description": "",
   "scripts": {
-    "test": "yarn forge test --no-match-path '*/upgrade/**.sol'",
+    "test": "yarn forge test --no-match-path '*/fork/**.sol'",
     "test:upgrade": "ts-node ./src/cli/upgrade/main.ts",
+    "test:fork": "yarn forge test --match-path '*/fork/**.sol'",
```

Comment:
So do we not actually run fork tests as part of CI?

Reply:
it will be finicky if the provider fails, depends on your tolerance for bad CI